gradio/external.py

```diff
@@ -334,6 +340,10 @@ def fn(*data):
                     try:
                         output = result["data"]
                     except KeyError:
```

Comment:
nope, if result.status_code == 429

Reply:
Unfortunately, the HF Hub API seems to return a 500 error with the following message: "Could not complete request to HuggingFace API, Error 429"

Reply:
So I don't think this is possible

Reply:
ok if HF API doesn't support this, nvm its fine

Reply:
I would still check for a `response.status_code == 500 and 429 in result["error"], and id